dev-support/hbase_nightly_source-artifact.sh

```diff
@@ -125,7 +125,7 @@ fi
 # See http://hbase.apache.org/book.html#maven.release
 
 echo "Maven details, in case our JDK doesn't match expectations:"
-mvn --version --offline | tee "${working_dir}/maven_version"
+mvn --threads=2 --version --offline | tee "${working_dir}/maven_version"
```

Comment:
There's no value in `--threads` for grabbing the version info, is there?

Reply:
Yeah. Dumb change. Undoing.